templates/docs/layouts/application.md

```diff
@@ -10,51 +10,62 @@ context:
 
 ### Structure
 
-The application layout is a full-screen grid (represented by a root element with `l-application` class) consisting of several areas (represented by elements with class names such as `l-navigation`, `l-main`, etc.). The application layout styling defines the positioning and size of all the areas on different screen sizes. An example of this would be, the navigation sidebar will be fully expanded on large screens, but collapsible on small screens.
+The application layout is a css grid based layout that occupies the entire window. Its root element has a class called `l-application`. The layout defines several areas represented by elements with class names such as `l-navigation`, `l-main`, etc.
 
-The minimal application layout should consist of the root element (`l-application`) and a navigation area (`l-navigation-bar`, `l-navigation`) with a main content area (`l-main`). The other areas (`l-aside` and `l-status`) are optional and may be added to provide additional information as required by given application use case.
+The application layout is responsive out of the box, aiming to provide a useable set of defaults. For example, the navigation sidebar is fully expanded on large screens, but collapses on small screens.
+
+The applicatino layout requires a root element (`l-application`), a navigation area (`l-navigation-bar`, `l-navigation`) and a main content area (`l-main`).
+
+It also includes optional sidebars (`l-aside`) and a status bar (`l-status`).
 
 #### Root element
 
-The root of application layout, usually a direct child of the document `body` is an element with `l-application` class. This element defines the layout grid, all areas of the application layout need to be direct children.
+The root of the application layout is usually a direct child of the document `body`. It is denoted with the class `l-application`. This element defines the layout grid. All areas of the application layout need to be direct children of this root element.
 
 #### Navigation
 
-To provide fully responsive behaviour for small and large screen navigation. The navigation of this layout consists of two areas.
+The navigation of this layout consists of two areas:
+
+An element with class `l-navigation-bar` which represents a horizontal navigation bar visible on small screens only. It toggles the navigation sidebar drawer. We recommend using the `<header>` tag for this element, as this ensures it acts as an <a href="https://www.w3.org/TR/wai-aria-practices/examples/landmarks/region.html">ARIA region landmark</a>. E.g.: `<header class="l-navigation">`.
```

Comment:
@lyubomir-popov This is not correct. The `<header>` should be used for `l-navigation`, not `l-navigation-bar`. Also, next paragraph says about `l-navigation__drawer` as a direct child - this is child of `l-navigation`, not `l-navigation-bar`.